features/admin.applications.v1/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -1132,20 +1134,29 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                         && isFAPIAppCreationEnabled
                         && (
                             <div className="pt-0 mt-0">
-                                <Field
-                                    data-componentid={ `${ testId }-fapi-app-checkbox` }
-                                    name={ "isFAPIApp" }
-                                    required={ false }
-                                    type="checkbox"
-                                    value={ [ "isFAPIApp" ] }
-                                    children={ [
-                                        {
-                                            label: t("applications:forms.generalDetails" +
-                                                ".fields.isFapiApp.label" ),
-                                            value: "fapiApp"
-                                        }
-                                    ] }
-                                />
+                                <div className="app-type-checkbox" >
+                                    <Field
+                                        data-componentid={ `${ testId }-fapi-app-checkbox` }
+                                        name={ "isFAPIApp" }
```

Comment:
We don't need to wrap string props with parenthesis.
```suggestion
                                        name="isFAPIApp"
```

Reply:
fixed with [0a6d32d](https://github.com/wso2/identity-apps/pull/6097/commits/0a6d32ddf50aa6a2eee55308cee0f45416d9844f)